apps/ai-pipeline/.env.example

```
@@ -33,18 +23,19 @@ IS_PRODUCTION=false
+MONGODB_URL=mongodb://sos_user:localpass123@localhost:27017/?authSource=admin
+MONGODB_DB=sos_db
 
 # 파이프라인 설정
 MAX_DEPTH=5
 MAX_CHOICES=3
 SEMAPHORE_LIMIT=5
 RETRY_COUNT=2
 LLM_TIMEOUT=60
-PIPELINE_TIMEOUT=3000
+# 큰 시나리오 트리 + 전체 이미지 생성까지 포함하므로 짧게 잡으면 timeout 가능.
+PIPELINE_TIMEOUT=12000
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The PIPELINE_TIMEOUT key should go before the RETRY_COUNT key

(UnorderedKey)